### ISSUE TITLE
Error on modify.inc.php to last release on Flyspray 1.0-rc11 ( and issue ? )

### ISSUE THREAD

**Issue by BeMichaelOlivier** [flyspray/flyspray#923]
Hello,

I wanted to report an issue with the file **modify.inc.php** in the latest release of **Flyspray 1.0-rc11**. When modifying categories, the following error message appears:

`Warning: foreach() argument must be of type array|object, null given in /var/www/html/includes/modify.inc.php on line 2234`

After checking the repository on GitHub, it seems that **the file in the release** archive flyspray-1.0-rc11.tgz **does not match the version in the GitHub repository**.

For me, To resolve the issue, I did the following:

- Deleted the modify.inc.php file from my installation.
- Downloaded the updated modify.inc.php file from the GitHub project.
- Uploaded and replaced the modify.inc.php file.

Since then, I’ve been able to move and delete categories without any issues.

Wishing you a great day.

Best regards,
M. Olivier

**Comment by Neustradamus** [comment]
@peterdd: Have you seen this ticket?

**Comment by Neustradamus** [comment]
@BeMichaelOlivier, it is time to recall @peterdd ^^

**Comment by peterdd** [comment]
Yes, category editing bug is a regression of 1.0-rc11 release (https://github.com/flyspray/flyspray/releases/tag/v1.0-rc11) and fixed in current master branch.
